packages/@tinacms/datalayer/src/node.ts

```diff
@@ -0,0 +1,154 @@
+import type { IncomingMessage, ServerResponse } from 'http'
```

Comment:
move `node.ts` to a subfolder like `backend` or `api`

Reply:
Moved all of the code from `node.ts` into `backend/index.ts`. The root `node.ts` will need to stay there and re-export all of the code to make the import path easier to understand. 

Ex: `import { TinaNodeBackend } from '@tinacms/datalayer/dist/node'`  instead of `import { TinaNodeBackend } from '@tinacms/datalayer/dist/backend/index'`

Reply:
I'm not sure I understand why we aren't just exporting `TinaNodeBackend` from index.ts so the import is `import { TinaNodeBackend } from '@tinacms/datalayer'`